packages/bpk-component-calendar/src/BpkCalendarGrid.js

```diff
@@ -91,7 +83,6 @@ class BpkCalendarGrid extends Component {
       dateProps,
       focusedDate,
       formatDateFull,
-      formatMonth,
```

Comment:
This has been removed but still is mentioned in the README is this intentional?

Reply:
Apologies, I will double check the README. It is the same case as before, it was used by the hidden components and not used anymore in the grid. Note that the rest of the calendar still uses this, but not the grid component.

Reply:
Double checked the README. I left the occurrences in `Calendar` and `CalendarNav`, as well as the explanation of this property, because it is still used in these two components. The reference from `CalendarGrid` have already been removed in my original commit. 😄